src/Plugins/Scrabble.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types=1);
+
+namespace Room11\Jeeves\Plugins;
+
+use Amp\Promise;
+use Room11\Jeeves\Chat\Client\ChatClient;
+use Room11\Jeeves\Chat\Message\Command;
+use Room11\Jeeves\System\PluginCommandEndpoint;
+
+class Scrabble extends BasePlugin
+{
+    const USAGE = "Usage: `!!scrabble [words to calculate score for]`";
+
+    const SCORES = [
+        'en' => [
+            1 => 'EAIONRTLSU',
+            2 => 'DG',
+            3 => 'BCMP',
+            4 => 'FHVWY',
+            5 => 'K',
+            8 => 'JX',
+            10 => 'QZ'
+        ]
+    ];
+
+    private $chatClient;
+
+    public function __construct(ChatClient $chatClient)
+    {
+        $this->chatClient = $chatClient;
+    }
+
+    public function getDescription(): string
+    {
+        return 'Calculates the Scrabble score for the given input';
+    }
+
+    public function scrabble(Command $command): Promise
+    {
+        if ($command->hasParameters() === false) {
```

Comment:
I don't think it matters whether it's the short form or explicit. We have both in the code base, so it doesn't need fixing.